.gitlab-ci.yml

```diff
@@ -7,8 +7,9 @@ before_script:
     - export OMP_NUM_THREADS=1
     - export CMAKE_VERSION=3.10.3
     - export CTEST_OUTPUT_ON_FAILURE=1
+    - export J=$(nproc)
     - wget -qO- http://www.cmake.org/files/v${CMAKE_VERSION:0:4}/cmake-${CMAKE_VERSION}-Linux-x86_64.tar.gz | tar -xz
-    - export PATH=$(pwd)/cmake-${CMAKE_VERSION}-Linux-x86_64/bin:$PATH
+    - export PATH=${PWD}/cmake-${CMAKE_VERSION}-Linux-x86_64/bin:$PATH
```

Comment:
Any particular reason for this change?

Reply:
`$(pwd)` is a command in a subshell, while `${PWD}` is just a variable expansion, that is slightly faster.